### ISSUE TITLE
Examples to be incorporated

### ISSUE THREAD

**Issue by elizhobart** [rbms-bsc/DCRMR#12]
Flagging for future work. Examples from the PS draft at 2.17.1.3 (Making Notes on Manifestation) map to specific elements and will likely need to be added there, including: Publication statement, Transcription (Clara Hayward example), font, physical description, edition. I want to make sure we don’t miss these as we finalize the text.

**Comment by elizhobart** [comment]
Likewise, will need to incorporate examples for multipart monographs and serials at 2.17.11.6.1.

**Comment by ladylazarus3** [comment]
yes, i think we will have to go through examples again with a fine tooth-comb once things are bit more developed and add more examples in as additional format modules come online.

Just so folks have a handy link-- there is guidance for [current best practices for formatting examples in Github Markdown](https://github.com/rbms-bsc/DCRMR/wiki/How-to-populate-a-new-Entity-Markdown-file#helpful-tips-for-formatting-examples-and-comments) in many of the "How to" documentation in the [DCRMR editorial wiki](https://github.com/rbms-bsc/DCRMR/wiki), but these are always changeable.  For now, i was just using the published DCRM manuals as a precedent with some modifications for web accessibility.

**Comment by mjmascaro** [comment]
In Edition Statement section "Two or more works on the preferred source of information with at least one edition statement"  current example of the Winnipeg & Manitoba street atlas; Manitoba back road atlas needs to be looked at make sure it is applicable.  The particular publication lacks a collective title page and the title are transcribed from the front and back cover.  (Okay to treat the whole cover as a collective source of information?)

**Comment by deborahjleslie** [comment]
@mjmascaro The atlas is a cartographic example. Use example in DCRMB? Although I would quibble with punctuation/capitalization.

>An examination of Dr. Burnet's theory of the earth ... / by J. Keill, The second edition corrected ... To the whole is annexed A dissertation on the different figures of coelestial bodies, &c ... / by Mons. de Maupertuis

**Comment by mjmascaro** [comment]
@deborahjleslie the DCRMB example has already been included in the text.  It illustrates the situation where the edition statement is grammatically linked, and therefore transcribed as part of another element.  

The atlas example (from the PS drafts) is an example of edition statements not being grammatically linked to other text, and therefore transcribed in repeating edition statement elements.  Agreed, not the best example (both due to being atlas and also the fact there is no collective title page), and if possible replaced in favor of another example that better illustrates this particular situation.
